homeyess/website/views.py

```diff
@@ -270,10 +273,23 @@ def get_context_data(self, *args, **kwargs):
         context['update'] = True
         return context
 
+    def form_valid(self, form):
```

Comment:
ik this isnt along the lines of this pr in particular, but when a hp edits their request, can you set the volunteer to None (as well as send the message)